remediation/remediation.md

```diff
@@ -5,16 +5,18 @@ nav_order: 4
 has_children: true
 ---
 
-# Putting Nodes into a Safe State (Remediation)
+## Putting Nodes into a Safe State (Remediation)
+
 {: .no_toc }
 
-## Table of contents
+### Table of contents
+
 {: .no_toc .text-delta }
 
 1. TOC
 {:toc}
 
-## The Problem
+### The Problem
 
 A common approach to for providing high availability for Kubernetes is to run
```

Comment:
Typo: remove "to"

Also, not sure about "providing" later in the paragraph, it does not fit with previous verb:

"to run ... and start ..."

Reply:
Nevermind, this is not part of the PR. Let's consider it a reminder for a future fix :)

Reply:
You can open up a PR for this manner :)